lib/rules/no-descending-specificity/index.mjs

```diff
@@ -151,6 +157,21 @@ function lastCompoundSelectorWithoutPseudoClasses(selectorNode) {
 	return nodesWithoutPseudoClasses.join('');
 }
 
+/**
+ * Specificity only has an effect on declarations.
+ * We only want to check rules that contain declarations either directly or in nested at-rules.
+ *
+ * @param {import('postcss').Container} node
+ * @returns {boolean}
+ */
+function hasDeclaration(node) {
+	if (!node.nodes) return false;
+
+	return node.some((child) => {
+		return isDeclaration(child) || (isAtRule(child) && hasDeclaration(child));
```

Comment:
question
minor

Is `from { margin-top: 50px; }` considered a declaration?
i.e. inside an `@keyframes`

Reply:
Yes, this is a declaration